exercism/lucians-luscious-lasagna.cs

```diff
@@ -0,0 +1,19 @@
+class Lasagna
+{
+    // TODO: define the 'ExpectedMinutesInOven()' method
+    public int ExpectedMinutesInOven(){
+        return 40;
+    }
+    // TODO: define the 'RemainingMinutesInOven()' method
+        public int RemainingMinutesInOven(int minutesInOven){
+            return ExpectedMinutesInOven() - minutesInOven;
+        }
+    // TODO: define the 'PreparationTimeInMinutes()' method
+        public int PreparationTimeInMinutes(int numberLayers){
+            return numberLayers * 2;
```

Comment:
It's a decent idea to avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants). If you name a [constant](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/constants), it helps it to be [self-documenting](https://en.wikipedia.org/wiki/Self-documenting_code). So you could consider, at the top of the class, doing this:

```cs
class Lasagna
{
    private const int MinutesPerLayer = 2;
// ...
```

Notice the `private` visibility: keeping things class private is a good idea because of the [Interface-Segregation Principle](https://en.wikipedia.org/wiki/Interface_segregation_principle) (it helps consumers of your class because they don't have extra stuff that could break their code, and it helps you because other people don't rely on stuff you might choose to change).